tests/__init__.py

```diff
@@ -110,8 +111,15 @@ def test_func(self):
                         msg=f"The actual value for .{key}() did not match the expected value.",
                     )
 
-        # Assert that the ingredients returned by the ingredient_groups() function
-        # are the same as the ingredients return by the ingredients() function.
+        # Special handling for ratings_count
+        if "ratings_count" in expect.keys():
+            with self.subTest("ratings_count"):
+                ratings_count = actual.ratings_count()
+                self.assertTrue(
+                    ratings_count <= expect["ratings_count"],
```

Comment:
This range-check might be inverted, I think?

It's valid for `actual_ratings_count` (count from the latest test HTML page) to be greater-than-or-equal to the `expected_ratings_count` (count from the test HTML page when the test expectations were first added / last updated).

Reply:
Sure was! It should be right now (i hope)